web/templates/settings_overlay.hbs

```diff
@@ -79,10 +79,8 @@
                     <li tabindex="0" data-section="emoji-settings">
                         <i class="icon fa fa-smile-o" aria-hidden="true"></i>
                         <div class="text">{{t "Custom emoji" }}</div>
-                        {{#if is_guest}}
-                            <i class="locked fa fa-lock" title="{{t 'Guest users cannot edit custom emoji.' }}"></i>
-                        {{else if show_emoji_settings_lock}}
-                            <i class="locked fa fa-lock" title="{{t 'Only organization administrators can edit these settings.'}}"></i>
+                        {{#if show_emoji_settings_lock}}
+                            <i class="locked fa fa-lock" title="{{t 'You do not have permission to add custom emoji.'}}"></i>
```

Comment:
I have not done similar change for the other lock icons for now because those are only shown to non-admins. But we can also update them for consistency